src/Common/Exceptions/DirectoryExistingException.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __DIRECTORY_EXISTING_EXCEPTION_H_
+#define __DIRECTORY_EXISTING_EXCEPTION_H_
+
+#include "BaseException.h"
+
+namespace FileManager {
+
+    class DirectoryExistingException : public BaseException {
+    public:
+
+        std::string path;
+
+        DirectoryExistingException(std::string path)
+        : BaseException("Directory " + path + " already exists.") {
```

Comment:
I am thinking to change the convention a little bit when specifying the initialization list ..
What do you think?

```suggestion
        : BaseException("Directory " + path + " already exists.")
        {
```

Reply:
Mmmm I actually prefer it to always keep the `{` on a new line. But anyway, I prefer it to be consistent with(out) the initialization list.